linked_lists/linked_list/linked_list.py

```diff
@@ -33,7 +33,7 @@ def append(self, data):
             return None
         node = Node(data)
         if self.head is None:
-            self.head = Node(data)
+            self.head = node
```

Comment:
Looks good, can we make these changes to the solution notebook instead of the .py file directly?  The solution notebook writes the .py file so both are in sync.